test/integration/targets/reboot/tasks/main.yml

```diff
@@ -37,11 +37,13 @@
     - name: Reboot with all options
       reboot:
         connect_timeout: 30
+        extra_search_paths: /usr/local/bin
```

Comment:
s/extra_// - and BTW, shouldn't this be failing tests -both because reboot isn't in /usr/local/bin and because `extra_search_paths` isn't a valid arg?

Reply:
Funny thing about these tests is they don't run due to the lack of separation of controller/target in tests. But I keep updating them and running them locally for when we _can_ use them. Evidently I failed to run them before pushing, though. :)